updater/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
+  id('java-library')
+  id('application')
+  alias(libs.plugins.spotless)
+}
+
+java {
+  toolchain {
+    languageVersion = JavaLanguageVersion.of(18)
```

Comment:
`17` instead? since it's the LTS release and some may not even have `18` installed.

Reply:
Gradle will automatically download a JDK to fulfill the toolchain if one is not detected on the system. The idea is that you can have any JDK (or JRE) on your system to invoke Gradle and then it will take care of provisioning that necessary one to build the project.

That being said, it defaults to Adoptium which have not completed their rollout of 18 🤦 so I'll drop it back down to 17 for now.